src/services/auth/applications/types.ts

```diff
@@ -188,6 +188,11 @@ export interface OAuth2Application extends Application {
   confidential?: boolean;
 }
 
+export enum OAuthApplicationType {
```

Comment:
(minor, probably not even worth fixing) `ApplicationType` would be enough in my opinion. I would even argue more correct. As applications might not just be an OAuth variation but in the future some mechanism like "JWT"/"Cookie"/"web"/...